src/lectures/models/intro_models.ipynb

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
<p>Would raw html in markdown be better here? https://jupyterbook.org/en/stable/advanced/html.html#use-raw-html-in-markdown</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/earth-env-data-science/earth-env-data-science-book/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Changed</p>